torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -60,21 +91,48 @@ def _get_state_dict(self, dl_kwargs):
         state_dict = load_state_dict_from_url(url, **dl_kwargs)
         return state_dict
 
-    def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
+    def get_model(self, *, dl_kwargs=None) -> Module:
         """Construct the model and load the pretrained weight.
 
         The weight file is downloaded from the internet and cached with
         :func:`torch.hub.load_state_dict_from_url`
 
         Args:
             dl_kwargs (dictionary of keyword arguments): Passed to :func:`torch.hub.load_state_dict_from_url`.
+
+        Returns:
+            Variation of :py:class:`~torchaudio.models.Wav2Vec2Model`.
+            For the following models:
+
+            WAV2VEC2_LARGE
+            WAV2VEC2_ASR_LARGE_10M
+            WAV2VEC2_ASR_LARGE_100H
+            WAV2VEC2_ASR_LARGE_960H
+            WAV2VEC2_LARGE_LV60K
+            WAV2VEC2_ASR_LARGE_LV60K_10M
+            WAV2VEC2_ASR_LARGE_LV60K_100H
+            WAV2VEC2_ASR_LARGE_LV60K_960H
+            WAV2VEC2_XLSR53
+            HUBERT_LARGE
+            HUBERT_XLARGE
+            HUBERT_ASR_LARGE
+            HUBERT_ASR_XLARGE
+            WAVLM_LARGE
```

Comment:
The documents can be improved, could you add blank lines between the bundles?
<img width="748" alt="image" src="https://user-images.githubusercontent.com/8653221/204540266-53744563-94d2-4c15-b6aa-cdfb17166d6d.png">

Reply:
> The documents can be improved, could you add blank lines between the bundles?

I suggest
 - use the list syntax
 - move the list to the end of the docstring for improved readability.
 - Not mention internal structure `_Wav2Vec2Model`.

something like the following

```
For the following models, an additional layer normalization is performed on the input.

- WAV2VEC2_LARGE
- WAV2VEC2_ASR_LARGE_10M
...
```